loadgen/helpers.go

```diff
@@ -0,0 +1,34 @@
+package loadgen
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.temporal.io/api/workflowservice/v1"
+	"go.temporal.io/sdk/client"
+)
+
+func VisibilityCountIsEventually(
```

Comment:
In the other PR, Roey was requesting documentation on all these public functions